src/libraries/System.Text.Json/src/System/Text/Json/Writer/JsonWriterOptions.cs

```diff
@@ -18,6 +18,8 @@ public struct JsonWriterOptions
         private int _maxDepth;
         private int _optionsMask;
 
+        internal readonly byte IndentByte => (_optionsMask & IndentCharacterBit) != 0 ? JsonConstants.Tab : JsonConstants.Space;
```

Comment:
Could this value be cached in Utf8JsonWriter instead of having to derive it every time indentation is being written?

Reply:
Yes, it should help to avoid regression around performance.

Reply:
For simplicity, I would recommend only keeping a `char` property and making the `byte` conversion in the `Utf8JsonWriter` layer on construction.

Reply:
OK.